code/game/gamemodes/marker/marker_shard.dm

```diff
@@ -107,15 +107,15 @@
 /obj/item/marker_shard/proc/activate()
 	active = TRUE
 	last_known_location = loc
-	GLOB.moved_event.register(src, src, /obj/item/marker_shard/moved)
```

Comment:
why this? The left is how i've always done event registration, it needs some explanation